exo-run/features/support/world.ls

```diff
@@ -24,4 +29,5 @@ World = !->
 
 
 module.exports = ->
+  @app-dir = app-dir
```

Comment:
Please make the comments meaningful. Regarding sharing the `@app-dir` variable, making it an instance variable (by prepending it with `@`) makes it available to all steps, no matter which file. Because `this` in a step is the same `world` instance. So you should be good with naming it `@app-dir` instead of `app-dir`, then it is shared everywhere automatically.